x-pack/test/alerting_api_integration/security_and_spaces/group5/config.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from '../../common/config';
+
+// eslint-disable-next-line import/no-default-export
+export default createTestConfig('security_and_spaces', {
+  disabledPlugins: [],
+  license: 'trial',
+  ssl: true,
+  enableActionsProxy: true,
+  publicBaseUrl: true,
+  testFiles: [require.resolve('./tests/alerting')],
```

Comment:
optional: for consistency with `group1` that was split and maybe in the future someone will have to re-balance it again we can add `index.ts` in `group5/tests` directory: 


```
export default function alertingApiIntegrationTests({ loadTestFile }: FtrProviderContext) {
  describe('alerting api integration security and spaces enabled', function () {
    loadTestFile(require.resolve('./alerting'));
  });
}
```

 and change this line to `  testFiles: [require.resolve('./tests')],`